variables.tf

```diff
@@ -172,6 +172,17 @@ variable "cluster_ec2_volume_size" {
   default     = 30
 }
 
+variable "metric_type" {
+  description = "Select CPU_UTILISATION to perform auto scaling based on CPU Utilisation, or select MEMORY_UTILISATION for MEMORY Utilisation."
+  type        = string
+  default     = "CPU_UTILISATION"
```

Comment:
Is it so, that I cannot disable autoscaling at all?

Reply:
In old logic, it was not possible but now it can be done from terra3.